### PR TITLE
husky code for PR

### PR DESCRIPTION
**Pull request by nickoenig37**
The following files allow for the 2D NAV2 stack to work on the husky as well as to move the husky with the remote.

### CODE REVIEW

File: husky/husky_base/launch/localization_launch.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
+# Copyright (c) 2018 Intel Corporation
```
Comment (MarcAntoineLavoie) [inline husky/husky_base/launch/localization_launch.py:1]: Is this used?

File: husky/husky_base/package.xml [hunk 2]
```diff
@@ -0,0 +1,35 @@
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
```
Comment (MarcAntoineLavoie) [inline husky/husky_base/package.xml:1]: needs cleanup

File: husky/husky_bringup/launch/accessories.launch.py [hunk 3]
```diff
@@ -0,0 +1,218 @@
+import os
```
Comment (MarcAntoineLavoie) [inline husky/husky_bringup/launch/accessories.launch.py:1]: Do we need this? We are using different sensors, no?

**Review by MarcAntoineLavoie — COMMENTED** [review]
Which of these files are different from the humble branch of the husky code? Why is this not a new repo? Same for pointcloud to laserscan, why is this copied here?

Remove the /.vscode files, they are specific to your machine

File: husky/README.md [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+git clone https://github.com/TRAILab/husky_trailbot.git
+git clone https://github.com/TRAILab/velodyne.git
+git clone https://github.com/TRAILab/ximea-driver.git
+cd ximea-driver
+./tools/install_sdk.sh
+cd ..
+
+```
+build
+```
```
Comment (barzanisar) [inline husky/README.md:42]: colcon build fails due to pcap.h not found. 

To install all package dependencies, run:
cd ~/trail_ws
rosdep install --from-paths src --ignore-src --rosdistro humble -r -y

To only install libpcap, run:
sudo -H apt-get install -y libpcap0.8-dev

File: husky/README.md [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+
+```
+
+
+Launch and Use Husky
+=============
+
+To run the needed launch file(s) to use the trailbot follow the steps below:
+
+Making sure that the husky is turned input:
```
Comment (barzanisar) [inline husky/README.md:58]: what does "turned input" mean?

**Review by barzanisar — CHANGES_REQUESTED** [review]
I got build errors when I followed the readme so some dependencies need to be installed before colcon build. I was able to launch the husky base on my laptop. I didn't review the rest of the code (cz I think it needs time to understand how nav2 works which I will do in the future).

File: husky/README.md [hunk 6]
```diff
@@ -0,0 +1,113 @@
...
+
+Install NAV2 and Correlating Dependencies:
+  - sudo apt install ros-humble-slam-toolbox
+  - sudo apt install ros-humble-navigation2 ros-humble-nav2-bringup
+  - sudo apt-get install ros-humble-turtlebot3*
+  - sudo apt install ros-humble-twist-mux
+  - sudo apt install ros-humble-gazebo-ros2-control
+
+Install this dependency from the top folder (i.e. "ros2_ws") in your directory
+  - cd ~/ros2_ws rosdep install --from-paths src --ignore-src --rosdistro humble -r -y
```
Comment (barzanisar) [inline husky/README.md:28]: There should be a new line between these two:
cd ~/ros2_ws 
rosdep install --from-paths src --ignore-src --rosdistro humble -r -y

**Review by barzanisar — APPROVED** [review]

**Pull request merged**